docs/language-usage.rst

```diff
@@ -0,0 +1,279 @@
+   API is not intended for real-time usage in critical applications.
+
+.. _Google Natural Language: https://cloud.google.com/natural-language/docs/getting-started
+
+Client
+------
+
+:class:`~gcloud.language.client.Client` objects provide a
+means to configure your application. Each instance holds
+both a ``project`` and an authenticated connection to the
+Natural Language service.
+
+For an overview of authentication in ``gcloud-python``, see
+:doc:`gcloud-auth`.
+
+Assuming your environment is set up as described in that document,
+create an instance of :class:`~gcloud.language.client.Client`.
+
+  .. code-block:: python
+
+     >>> from gcloud import language
+     >>> client = language.Client()
+
+By default the ``language`` is ``'en'`` and the ``encoding`` is
+UTF-8. To over-ride these values:
+
+  .. code-block:: python
+
+     >>> client = language.Client(language='es',
+     ...                          encoding=encoding=language.Encoding.UTF16)
+
+The encoding can be one of
+:attr:`Encoding.UTF8 <gcloud.language.document.Encoding.UTF8>`,
+:attr:`Encoding.UTF16 <gcloud.language.document.Encoding.UTF16>`, or
+:attr:`Encoding.UTF32 <gcloud.language.document.Encoding.UTF32>`.
+
+Methods
+-------
+
+The Google Natural Language API has three supported methods
+
+- `analyzeEntities`_
+- `analyzeSentiment`_
+- `annotateText`_
+
+and each method uses a `Document`_ for representing text. To
+create a :class:`~gcloud.language.document.Document`,
+
+  .. code-block:: python
+
+     >>> text_content = (
+     ...     'Google, headquartered in Mountain View, unveiled the '
+     ...     'new Android phone at the Consumer Electronic Show.  '
+     ...     'Sundar Pichai said in his keynote that users love '
+     ...     'their new Android phones.')
+     >>> document = client.document_from_text(text_content)
+
+By using :meth:`~gcloud.language.client.Client.document_from_text`,
+the document's type is plain text:
+
+  .. code-block:: python
+
+     >>> document.doc_type == language.Document.PLAIN_TEXT
+     True
+
+In addition, the document's language defaults to the language on
+the client
+
+  .. code-block:: python
+
+     >>> document.language
+     'en'
+     >>> document.language == client.language
+     True
+
+In addition, the
+:meth:`~gcloud.language.client.Client.document_from_html`,
+factory can be used to created an HTML document. In this
+method and the from text method, the language can be
+over-ridden:
+
+  .. code-block:: python
+
+     >>> html_content = """\
+     ... <html>
+     ...   <head>
+     ...     <title>El Tiempo de las Historias</time>
+     ...   </head>
+     ...   <body>
+     ...     <p>La vaca salt&oacute; sobre la luna.</p>
+     ...   </body>
+     ... </html>
+     ... """
+     >>> document = client.document_from_html(html_content,
+     ...                                      language='es')
+
+The ``language`` argument can be either ISO-639-1 or BCP-47 language
```

Comment:
Can we focus on BCP-47 rather than ISO-639-1 (that's the direction the APIs will be going)